packages/google_maps_flutter/google_maps_flutter_web/lib/src/google_maps_controller.dart

```diff
@@ -41,14 +41,14 @@ class GoogleMapController {
   }
 
   // The currently-enabled traffic layer.
-  gmaps.TrafficLayer _trafficLayer;
+  gmaps.TrafficLayer? _trafficLayer;
 
   /// A getter for the current traffic layer. Only for tests.
   @visibleForTesting
-  gmaps.TrafficLayer get trafficLayer => _trafficLayer;
+  gmaps.TrafficLayer? get trafficLayer => _trafficLayer;
 
   // The underlying GMap instance. This is the interface with the JS SDK.
-  gmaps.GMap _googleMap;
+  gmaps.GMap? _googleMap;
```

Comment:
`late`?

Reply:
No, I want to clear `gmaps.GMap` on `dispose`, same as with the other geometry objects.

Reply:
I've added a note to dispose that you can't call "many of the methods" of the controller after `dispose`. However this code is used through the gmaps plugin, and not directly by end users.